version/version.go

```diff
@@ -1,13 +1,47 @@
 package version
 
+import (
+	"runtime/debug"
+	"strconv"
+)
+
+const (
+	defaultVersion = "0.0.0+unknown"
```

Comment:
nit: I think we should have had a proper suffix for the default version:

```suggestion
	defaultVersion = "v0.0.0+unknown"
```

Reply:
You're right :+1:

We should also apply a similar fix in buildkit imo.

Reply:
needs rebase since #1670